### PR TITLE
Eliminate teacher performance duplicate items

### PR DESCRIPTION
**Pull request by gsvr**
https://praekelt.atlassian.net/browse/RZ-19?jql=project%20%3D%20RZ%20AND%20resolution%20%3D%20Unresolved%20AND%20priority%20%3D%20Critical%20ORDER%20BY%20key%20DESC

### CODE REVIEW

**Comment by gsvr** [comment]
@imsickofmaps Please review.  I tested on QA line for teacher and learner performance, and both seem fine.  Not sure if we need to add tests.

File: js_sandbox/lib/go-rts-zambia.js [hunk 1]
```diff
@@ -1454,29 +1425,28 @@ function GoRtsZambia() {
             return self.check_valid_number(content);
         },
         "Please provide a number value for the pupils in the class that have broken " +
-            "through/can read."
+            "through/can read.",
+        {
+            on_exit: function() {
+                var p = self.cms_performance_teacher(im);
```
Comment (imsickofmaps) [inline js_sandbox/lib/go-rts-zambia.js]: You can just `return self.cms_performance_teacher(im);`

File: js_sandbox/lib/go-rts-zambia.js [hunk 2]
```diff
@@ -1764,30 +1728,30 @@ function GoRtsZambia() {
             return self.check_valid_number(content) && (parseInt(content,10) >= 0 && parseInt(content,10) <= parseInt(im.get_user_answer('perf_learner_girls_total'),10));
         },
         "Please provide a valid number value for total girls achieving 2 out of 4" +
-        " correct answers for Writing."
+        " correct answers for Writing.",
+        // Log the users data
+        {
+            on_exit: function() {
+                var p = self.cms_performance_learner(im);
```
Comment (imsickofmaps) [inline js_sandbox/lib/go-rts-zambia.js]: Same here

**Comment by imsickofmaps** [comment]
Minor comment then :+1: and land

**Pull request merged**